tests/driver.c

```diff
@@ -5,12 +5,25 @@
 static ta_core_t ta_core;
 struct timespec start_time, end_time;
 
+char DRIVER_TAG_MSG[NUM_TRYTES_TAG];
+send_mam_res_t* res;
+
 #if defined(ENABLE_STAT)
 #define TEST_COUNT 100
 #else
 #define TEST_COUNT 1
 #endif
 
+static void gen_rand_tag(char* tag) {
+  const char tryte_alpahbet[] = "NOPQRSTUVWXYZ9ABCDEFGHIJKLM";
+
+  srand(time(NULL));
```

Comment:
Why did you invoke `srand` here?

Reply:
I produce a random tag here for the usage of api_send_transfer().
so I use `srand` to produce random number

Reply:
`srand` should be invoked in the very beginning of test entries.

Reply:
got it